server/src/dtos/person.dto.ts

```diff
@@ -71,6 +71,7 @@ export class PersonResponseDto {
   birthDate!: Date | null;
   thumbnailPath!: string;
   isHidden!: boolean;
+  updatedAt!: Date;
```

Comment:
This is a breaking change for mobile. You should make it optional, and add the PropertyLifecycle() decorator to track when it was added.